### PR TITLE
Create PULL_REQUEST_TEMPLATE.md

### PR DESCRIPTION
**Pull request by inuyasha1205**
Test pull 2

### CODE REVIEW

**Comment by Tsearo** [comment]
Let's test editing a pull if possible

File: PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+Fixes #
+
+Changes proposed in this pull request:
+-
+-
+-
+
```
Comment (Tsearo) [inline PULL_REQUEST_TEMPLATE.md]: Add another '-' line

**Review by Tsearo — CHANGES_REQUESTED** [review]

**Comment by inuyasha1205** [comment]
Done

**Review by Tsearo — APPROVED** [review]
Looks good now

**Pull request merged**